content/learning-paths/smartphones-and-mobile/android_webgpu_dawn/6-render-a-simple-3D-object-part-2.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Render a simple 3D object - part 2
+weight: 7
+
+### FIXED, DO NOT MODIFY
+layout: learningpathall
+---
+
+## 3D meshes
+
+Once Render Pipeline is created, using WebGPU APIs to create and render a 3D mesh is very similar to other graphics APIs. The flow follows:
```

Comment:
You didn't really explain how to create the render pipeline once we had the shader module, though.

Reply:
The file was not saved properly, updated